Fabric.Identity.API/Services/ClaimsService.cs

```diff
@@ -136,8 +155,9 @@ private ClaimsResult GenerateNewClaimsResult(AuthenticateInfo info, Authorizatio
             };
         }
 
-        private Claim[] GenerateAdditionalClaims(List<Claim> previousClaims)
+        private Claim[] GenerateAdditionalClaims(ClaimsResult claimsResult, AuthenticateInfo authenticateInfo)

```

Comment:
AuthenticateInfo authenticateInfo parameter is not used.  Is this required?  Maybe we remove the need for this?